### DISCUSSION TITLE
你好，感觉反馈，可以参考这个pr进行修复：https://github.com/PaddlePaddle/PaddleOCR/pull/7586/files

### DISCUSSION THREAD

**Discussion by liuchong540** [PaddlePaddle/PaddleOCR#12272] [Q&A]
Traceback (most recent call last):
  File "tools/export_model.py", line 289, in <module>
    main()
  File "tools/export_model.py", line 284, in main
    export_single_model(
  File "tools/export_model.py", line 74, in export_single_model
    shape=[None] + input_shape, dtype="float32"),
TypeError: can only concatenate list (not "NoneType") to list
 
请问这个应该怎样解决啊

**Comment by UserWangZz** [comment]
可以检查一下，拼接操作的变量是否正常，可能有未实例化的对象

**Comment by zhangyubo0722** [comment]
麻烦提供一下复现命令吧

**Comment by Cupcc** [comment]
参考这个，使用2.6发行版的export_model.py文件，但我不知为什么这可以成功。
https://github.com/PaddlePaddle/PaddleOCR/issues/10621#issuecomment-1684056122
